cmd/root.go

```diff
@@ -108,6 +109,7 @@ func init() {
 	rootCmd.Flags().StringVar(&caFile, "ca-file", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
 	rootCmd.Flags().StringVar(&certFile, "cert-file", "", "identify HTTPS client using this SSL certificate file")
 	rootCmd.Flags().StringVar(&keyFile, "key-file", "", "identify HTTPS client using this SSL key file")
+	rootCmd.Flags().StringVar(&newChartHost, "overwrite-host", "", "overwrites the host in chart's dowload URLs of the index file")
```

Comment:
I think this name is confusing. I initially assumed that was the host name (eg: `chart-mirror.airgap.lan`), instead it's supposed to be the url of the root of the new repository. Maybe we should do something like:

```go
rootCmd.Flags().StringVar(
  &newRootUrl,
 "new-root-url",
  "",
 "New root url of the chart repository (eg: `https://mirror.local.lan/charts`")
```

Reply:
Most important of all: we should add some checks to ensure that the new url, if specified, is a valid one.